.github/workflows/openapi_validate.yaml

```diff
@@ -14,8 +14,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 15.x
-####### rhoas-list-schema skipped due to https://issues.redhat.com/browse/MGDOBR-935
-      - run: |
-          npx @stoplight/spectral@5.9.0 lint \
-          ./openapi/smartevents_mgmt.yaml \
-          --skip-rule rhoas-list-schema
\ No newline at end of file
+      - name: Install RHOAS Guidelines
+        run: npm i @rhoas/spectral-ruleset@0.3.0
+      - name: Validate OAS file
+        run: npx @rhoas/spectral-ruleset@0.3.0 lint ./openapi/smartevents_mgmt.yaml
```

Comment:
Could you please explain, why we are not using the latest, i.e. 0.3.1? https://www.npmjs.com/package/@rhoas/spectral-ruleset/v/0.3.1

Reply:
great catch, the latest was really just a doc change. cosmetic really. I've removed the explicit version. should pull `latest` now. 👍